logging.tf

```diff
@@ -1,7 +1,7 @@
 resource "aws_iam_role_policy" "instance" {
   count  = var.enable_cloudwatch_logging && var.create_runner_iam_role ? 1 : 0
   name   = "${local.name_iam_objects}-logging"
-  role   = local.aws_iam_role_instance_name
+  role   = var.create_runner_iam_role ? aws_iam_role.instance[0].name : var.runner_iam_role_name
```

Comment:
suggestion: replace `var.runner_iam_role_name`by `local.aws_iam_role_instance_name` in all changes.

The local variable respects the `var.overrides["name_iam_objects"]` which was introduced to align the names of IAM objects. The variable description says: `If unspecified then `{name_iam_objects}-instance` is used`.